src/main/flight/pid.c

```diff
@@ -96,21 +96,21 @@ PG_REGISTER_WITH_RESET_TEMPLATE(pidConfig_t, pidConfig, PG_PID_CONFIG, 2);
 #ifndef DEFAULT_PIDS_ROLL
 #define DEFAULT_PIDS_ROLL \
     {                     \
-        50, 70, 28, 0     \
+        50, 70, 28        \
```

Comment:
superflous spaces please consider using editor-side formatting at least

Reply:
not superflous if you look at the "\" character. I could the entire formatting or keep it this way. Either way i'm fine with.

Reply:
this comment was not related to the approval as the symbols next to it indicate

Reply:
i prefer the older style anyhow. i'll be putting it back to how it used to be.